Installation with mac M1

Hello,
I'm new to libmesh and I'm trying to make the local installation (mac M1 with OS Ventura 13.5) following the instructions. However, the installation fails during the `make` stage and I cannot figure out why. This is my command for the `configure`:
`./configure --prefix=/Users/crpiceda/libmesh/libmesh/install --disable-metaphysicl \`
The `config.log` is attached.

`make` is aborting after these lines: 
```
ld: warning: -bind_at_load is deprecated on macOS
make[1]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build'
```

and then if i run `make install`, I get this in the terminal:
```
Making install in include
make[1]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include'
Making install in libmesh
make[2]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make  install-am
make[3]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make[4]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make[4]: Nothing to be done for 'install-exec-am'.
make[4]: Nothing to be done for 'install-data-am'.
make[4]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make[3]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make[2]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include/libmesh'
make[2]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include'
make[3]: Entering directory '/Users/crpiceda/libmesh/libmesh/build/include'
make[3]: Nothing to be done for 'install-exec-am'.
 ../../build-aux/install-sh -c -d '/Users/crpiceda/libmesh/libmesh/install/include/libmesh'
mkdir: /Users/crpiceda/libmesh/libmesh/install: Not a directory
mkdir: /Users/crpiceda/libmesh/libmesh/install: Not a directory
make[3]: *** [Makefile:1139: install-includeHEADERS] Error 1
make[3]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include'
make[2]: *** [Makefile:1333: install-am] Error 2
make[2]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include'
make[1]: *** [Makefile:1166: install-recursive] Error 1
make[1]: Leaving directory '/Users/crpiceda/libmesh/libmesh/build/include'
make: *** [Makefile:32290: install-recursive] Error 1
```

Could you guide me in what I might be doing wrong? I'm running with the compilers installed via `homebrew`. I'm interested in using the exodus library.
Thanks in advance

[config.log](https://github.com/libMesh/libmesh/files/13565826/config.log)

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ The default is to build libmesh "out of tree," i.e. within a separate `build` di
 1. `make install`
 
 Afterward, `make uninstall` is available if needed to remove the files
-copied to /some/final/install/directory (and its subdirectories) by
+copied to `/some/final/install/directory` (and its subdirectories) by
 `make install`.
 
 ## METHODS

